extensions/resteasy-common/runtime/src/main/java/io/quarkus/resteasy/common/runtime/MethodFullPathIndex.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.resteasy.common.runtime;
+
+import java.lang.reflect.Method;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class MethodFullPathIndex {
+    private static ConcurrentHashMap<Method, String> methodToFullPath = new ConcurrentHashMap<>();
```

Comment:
Can we make this an @Singleton bean instead? If not you will need to clear the map on undeploy otherwise you will have a ClassLoader leak in dev mode.

Reply:
Yes. We absolutely can.